File: PHP/Motorcycle.php
<?php
require_once 'Vehicle.php';

class Motorcycle extends Vehicle {
    private $type;
    private $tankCapacity;

    public function __construct($plateNumber, $brand, $year, $color, $type, $tankCapacity) {
        parent::__construct($plateNumber, $brand, $year, $color);
        $this->type = $type;
        $this->tankCapacity = $tankCapacity;
    }

    public function displayInfo() {
        parent::displayInfo();
        echo "Type: " . $this->type . PHP_EOL;
        echo "Tank Capacity: " . $this->tankCapacity . " liters" . PHP_EOL;
    }
}
?>

File: Motorcycle.java
package Java;

public class Motorcycle extends Vehicle {
    private String type;
    private int tankCapacity;

    public Motorcycle(String plateNumber, String brand, int year, String color, String type, int tankCapacity) {
        super(plateNumber, brand, year, color);
        this.type = type;
        this.tankCapacity = tankCapacity;
    }

    @Override
    public void displayInfo() {
        super.displayInfo();
        System.out.println("Type: " + type);
        System.out.println("Tank Capacity: " + tankCapacity + " liters");
    }
}

